cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -566,21 +567,21 @@ class EfsIdValidator(Validator):  # TODO add tests
     Validate if there are existing mount target in the head node availability zone
     """
 
-    def _validate(self, efs_id, head_node_avail_zone: str):
+    def _validate(self, efs_id, head_node_avail_zone: str, are_all_security_groups_customized):
         # Get head node availability zone
         head_node_target_id = AWSApi.instance().efs.get_efs_mount_target_id(efs_id, head_node_avail_zone)
         # If there is an existing mt in the az, need to check the inbound and outbound rules of the security groups
         if head_node_target_id:
             # Get list of security group IDs of the mount target
             sg_ids = AWSApi.instance().efs.get_efs_mount_target_security_groups(head_node_target_id)
-            if not _check_in_out_access(sg_ids, port=2049):
+            if not _check_in_out_access(sg_ids, port=2049, is_cidr_optional=are_all_security_groups_customized):
```

Comment:
by looking at this call it seems to me that we could rename the parameter to: `check_cidr` or `skip_cidr_check`.

I mean we're calling a function called `_check_in_out_access` and we can say to skip or not the cidr check.


```python
_check_in_out_access(sg_ids, port=2049, check_cidr=not are_all_security_groups_customized)
```
or
```python
_check_in_out_access(sg_ids, port=2049, skip_cidr_check=are_all_security_groups_customized)
```

anyway up to you, I approved the patch.